.github/workflows/stale-issue.yml

```diff
@@ -0,0 +1,100 @@
+            This issue has been automatically closed due to inactivity. If you can reproduce this on a
+            recent version of Gradle or if you have a good use case for this feature, please feel free
+            to to let know so we can reopen the issue. Please try to provide steps to reproduce, a quick
+            explanation of your use case or a high-quality pull request.
+          close-issue-reason: not_planned
+
+          # PULL REQUESTS -----------------------------------------------------
+          days-before-pr-stale: 60
+          stale-pr-label: stale
+          stale-pr-message: >
+            This pull request has been automatically marked as stale because it has not had recent activity.
+            Given the limited bandwidth of the team, it will be closed if no further activity occurs.
+            If you intend to work on this pull request, please ask the team to reopen the PR or push a new PR.
+            Thank you for your contributions.
+          days-before-pr-close: 21
+          close-pr-message: >
+            This pull request has been automatically closed due to inactivity.
+            If you are still interested in contributing this, please ensure that
+            it is rebased against the latest branch (usually `master`), all review
+            comments have been addressed and the build is passing.
+  stale-community-pr:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v5
+        with:
+          # GLOBAL ------------------------------------------------------------
+          # !!! Debug is on until stabilized
+          operations-per-run: 100
+          only-labels: 'from:contributor'
+
+          # ISSUES (deactivated) ----------------------------------------------
+          # This workflow should touch no issues, but actions/stale has no only-pr option.
+          # In order to deactivate this part, the following is done:
+          #  - Large stale time
+          #  - Only 'from:contributor' labels: this label should only appear oo PRs, and not on issues
+          #  - Exempt 'from:contributor' labels: forms a mutual exclusion with the only filter
+          #  - A bug message, so if the previous points stop working, it's obvious what happened
+          days-before-issue-stale: 3650
+          exempt-issue-labels: 'from:contributor'
+          stale-issue-label: stale
+          stale-issue-message: >
+            **BUG!** This issue should not be marked stale by the "stale-community-pr" workflow.
+            Please report it to @bt-developer-productivity
+          days-before-issue-close: 3650
+          close-issue-message: >
+            **BUG!** This issue should not be closed by the "stale-community-pr" workflow.
+            Please report it to @bt-developer-productivity
+          close-issue-reason: not_planned
+
+          # PULL REQUESTS -----------------------------------------------------
+          days-before-pr-stale: 365
```

Comment:
Notified @gradle/bt-support about this.
This is a pretty high stale time, so we don't cause any trouble even if this is merged.